src/beautifiers/beautysh.coffee

```diff
@@ -0,0 +1,15 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class BashBeautify extends Beautifier
+  name: "beautysh"
+  link: "https://github.com/bemeurer/beautysh"
+  options: {
+    Bash:
+      indent_size: true
```

Comment:
I apologize for this.

Could you also change package.json `scripts`:
- change `postinstall` key to `build-options`
- add `postinstall` script with value `npm run build-options`
- change `docs` script value to `npm run build-options && coffee docs/`

This should make sure `npm run docs` uses up-to-date `options.json`. Sorry about that!